src/jmcomic/jm_feature.py

```
@@ -0,0 +1,172 @@
+    download_album(id, option, extra=Feature.export_pdf)
+
+    # 带自定义参数
+    download_album(id, option, extra=Feature.export_pdf(pdf_dir='./output'))
+
+    # 多个 Feature（列表 / 运算符均可）
+    download_album(id, option, extra=[Feature.export_pdf, Feature.export_zip])
+    download_album(id, option, extra=Feature.export_pdf + Feature.export_zip)
+"""
+from typing import LiteralString
+
+from .jm_plugin import *
```

Ruff (0.15.12)

[error] `from .jm_plugin import *` used; unable to detect undefined names

(F403)